pegasus/sites.v3/code.org/public/educate/csp.md

```diff
@@ -1,17 +1,33 @@
 ---
 title: Computer Science Principles
 nav: educate_nav
+video_player: true
 ---
 
 # Computer Science Principles #
 
 ## <a name="whatis"></a>What is CS Principles?
-In fall 2016, the College Board will launch its newest AP® course, AP Computer Science Principles. The course introduces students to the foundational concepts of computer science and challenges them to explore how computing and technology can impact the world. The AP Program designed AP Computer Science Principles with the goal of creating leaders in computer science fields and attracting and engaging those who are traditionally underrepresented with essential computing tools and multidisciplinary opportunities.
 
-<iframe width="560" height="315" src="//www.youtube.com/embed/S1vFrz4NETg" frameborder="0" allowfullscreen></iframe>
+[col-50]
+
+In fall 2016, the College Board will launch its newest AP® course, AP Computer Science Principles. The course introduces students to the foundational concepts of computer science and challenges them to explore how computing and technology can impact the world. The AP Program designed AP Computer Science Principles with the goal of creating leaders in computer science fields and attracting and engaging those who are traditionally underrepresented with essential computing tools and multidisciplinary opportunities.
```

Comment:
I think html codes are probably a safer bet for wide compatibility, but for hotfix as-is should be fine.